src/lib/utils.js

```diff
@@ -92,20 +114,16 @@ const setLocalConfig = async (localConfig, actDir) => {
 
 const setLocalEnv = async (actDir) => {
     // Create folders for emulation Apify stores
-    const localDir = createFolderSync(path.join(actDir, DEFAULT_LOCAL_EMULATION_DIR));
-    const datasetsDir = createFolderSync(path.join(localDir, LOCAL_EMULATION_SUBDIRS.datasets));
-    const requestQueuesDir = createFolderSync(path.join(localDir, LOCAL_EMULATION_SUBDIRS.keyValueStores));
-    const keyValueStoresDir = createFolderSync(path.join(localDir, LOCAL_EMULATION_SUBDIRS.requestQueues));
-    createFolderSync(path.join(datasetsDir, DEFAULT_LOCAL_STORES_ID));
-    createFolderSync(path.join(requestQueuesDir, DEFAULT_LOCAL_STORES_ID));
-    createFolderSync(path.join(keyValueStoresDir, DEFAULT_LOCAL_STORES_ID));
+    ensureFolderExistsSync(actDir, getLocalDatasetPath());
+    ensureFolderExistsSync(actDir, getLocalRequestQueuePath());
+    ensureFolderExistsSync(actDir, getLocalKeyValueStorePath());
 
     // Update gitignore
     const gitingore = path.join(actDir, '.gitignore');
     if (fs.existsSync(gitingore)) {
-        fs.writeFileSync(gitingore, `\n${DEFAULT_LOCAL_EMULATION_DIR}`, { flag: 'a' });
+        fs.writeFileSync(gitingore, `\n${LOCAL_ENV_VARS[DEFAULT_LOCAL_STORAGE_DIR]}`, { flag: 'a' });
```

Comment:
It should be just `DEFAULT_LOCAL_STORAGE_DIR`, not `LOCAL_ENV_VARS[DEFAULT_LOCAL_STORAGE_DIR]`. Or maybe better `getLocalStorageDir()`